build/template-Build-run-tests-sign.yml

```diff
@@ -212,7 +212,7 @@ steps:
     PathtoPublish: '$(Build.SourcesDirectory)\artifacts'
     ArtifactName: '$(Build.BuildNumber)-nuget-package'
 
- - task: BuildQualityChecks@9
```

Comment:
Do you understand how this was working before? Or was it flaky?

I'm having a hard time understanding how this space change could have worked before if it was wrong, or, if it worked before, how removing the space now leads to it working. Can the description add more information on why this fixes the issue you encountered?

Reply:
Added in the origin comment